pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -2603,6 +2607,109 @@ func (m *UpdateRuntimeConfigResponse) String() string            { return proto.
 func (*UpdateRuntimeConfigResponse) ProtoMessage()               {}
 func (*UpdateRuntimeConfigResponse) Descriptor() ([]byte, []int) { return fileDescriptorApi, []int{71} }
 
+// RuntimeCondition contains condition information for the runtime.
+// There are 2 kinds of runtime conditions:
+// 1. Required condtitions: Conditions are required for kubelet to work
+// properly, including:
+//   * RuntimeReady: RuntimeReady means the runtime is up and ready to accept
+//   basic containers.
+//   * NetworkReady: NetworkReady means the runtime network is up and ready to
```

Comment:
so if any required condition is unmet the node remains notready (please elaborate comment)? 
we will probably have to conditionally start hostnetwork pods if NetworkReady == false but everything else checks out.